src/python/grpcio/grpc/__init__.py

```diff
@@ -858,6 +858,21 @@ def set_details(self, details):
     """
         raise NotImplementedError()
 
+    @abc.abstractmethod
+    def abort(self, code, details):
+        """Terminates the current RPC with a non-OK by raising an exception
+    status code.
+
+    The code and details passed as arguments will supercede the existing ones.
```

Comment:
This should be eight spaces in from the left edge. `pyformat` wasn't able to correctly adjust our doc strings when we jumped from two-space indents to four-space indents, but take a look at the way I recently cleaned up the credentials classes' doc strings as an example.

Reply:
Done. This needs a file-wide cleanup at some point.